src/lib/components/ChapterSelector.svelte

```diff
@@ -48,26 +48,28 @@ The navbar component.
 </script>
 
 <!-- Chapter Selector -->
-<Dropdown>
-    <svelte:fragment slot="label">
-        {$refs.chapter}
-        <DropdownIcon />
-    </svelte:fragment>
-    <svelte:fragment slot="content">
-        <TabsMenu
-            bind:this={chapterSelector}
-            options={{
-                Chapter: {
-                    component: SelectGrid,
-                    props: { options: Object.keys(chapters) }
-                },
-                Verse: {
-                    component: SelectGrid,
-                    props: { options: Object.keys(chapters[nextRef.chapter]) }
-                }
-            }}
-            active="Chapter"
-            on:menuaction={navigateReference}
-        />
-    </svelte:fragment>
-</Dropdown>
+{#if $globalConfig.mainFeatures['show-chapter-number-on-app-bar']}
+    <Dropdown>
```

Comment:
What about this setting:

`mainFeatures['show-chapter-selector']` - controls whether the chapter can be changed by a selector if the number displays on the app bar